app/questionnaire/questionnaire_schema.py

```diff
@@ -83,6 +84,29 @@ def get_answers_by_answer_id(self, answer_id):
         """
         return self._answers_by_id.get(answer_id)
 
+    def get_default_answer(self, answer_id):
+        try:
+            block = self.get_block_for_answer_id(answer_id)
+        except TypeError:
+            return None
+
+        try:
+            question = block['question']
+        except KeyError:
+            question = block['question_variants'][0]['question']
+
+        for this_answer in question['answers']:
+            if this_answer['id'] == answer_id:
+                answer = this_answer
```

Comment:
Can the variable `answer` be called something else because it is reassigned again below? Maybe `answer_schema`?